### PR TITLE
fix: Add translation hooks on remaining files 

### PR DESCRIPTION
**Pull request by iamsaumya**
towards #2117

### CODE REVIEW

File: app/scenes/Invite.js [hunk 1]
```diff
@@ -1,213 +1,223 @@
...
+          <Trans
+            defaults="Invite team members to join your knowledge base. They will need to
+          sign in with {{signinMethods}}."
+            values={{ signinMethods: team.signinMethods }}
+          />{" "}
+          {can.update && (
+            <Trans>
+              As an admin you can also{" "}
+              <Link to="/settings/security">enable email sign-in</Link>.
+            </Trans>
```
Comment (iamsaumya) [inline app/scenes/Invite.js:131]: Picked this from https://react.i18next.com/latest/trans-component#using-with-react-components

Comment (tommoor) [reply app/scenes/Invite.js:131]: 👍 Yep, this is fine when there is no interpolation

File: app/components/InputRich.js [hunk 2]
```diff
@@ -1,58 +1,58 @@
...
-  @observable editorComponent: React.ComponentType<any>;
-  @observable focused: boolean = false;
+function InputRich({ label, minHeight, maxHeight, ...rest }: Props) {
+  const [focused, setFocused] = React.useState<boolean>(false);
+  const { ui } = useStores();
 
-  handleBlur = () => {
-    this.focused = false;
-  };
+  const handleBlur = React.useCallback(() => {
```
Comment (tommoor) [inline app/components/InputRich.js:22]: There are so many places where we do this, need to introduce a `useBoolean` hook to cut down on boilerplate soon

File: shared/i18n/locales/en_US/translation.json [hunk 3]
```diff
@@ -335,6 +345,18 @@
   "This group has no members.": "This group has no members.",
   "Recently viewed": "Recently viewed",
   "Created by me": "Created by me",
+  "We sent out your invites!": "We sent out your invites!",
+  "Sorry, you can only send {{MAX_INVITES}} invites at a time": "Sorry, you can only send {{MAX_INVITES}} invites at a time",
+  "Invite team members or guests to join your knowledge base. Team members can sign in with {{signinMethods}} or use their email\n          address.": "Invite team members or guests to join your knowledge base. Team members can sign in with {{signinMethods}} or use their email\n          address.",
```
Comment (tommoor) [inline shared/i18n/locales/en_US/translation.json:350]: As in other PR, please tweak to remove newlines where not neccessary, thank you

**Review by tommoor — APPROVED** [review]

**Pull request merged**